changelog/dev-7438-payment-cart-supported-filter

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Introduce filter `wcpay_payment_request_is_cart_supported`.
```

Comment:
could you elaborate a little on the purpose of this filter?

Reply:
Expanded the detail in fc5ee98a